src/it/scala/com/databricks/spark/redshift/RedshiftIntegrationSuite.scala

```diff
@@ -263,21 +263,30 @@ class RedshiftIntegrationSuite
     // scalastyle:on
   }
 
-  test("DefaultSource using 'query' supports user schema, pruned and filtered scans") {
-    // scalastyle:off
-    QueryTest.checkAnswer(
-      sqlContext.sql(
-        """
-          |select testbyte, testbool
-          |from test_table
-          |where testbool = true
-          | and teststring = "Unicode's樂趣"
-          | and testdouble = 1234152.12312498
-          | and testfloat = 1.0
-          | and testint = 42
-        """.stripMargin),
-      Seq(Row(1, true)))
-    // scalastyle:on
+  test("roundtrip save and load") {
+    val tableName = s"roundtrip_save_and_load_$randomSuffix"
+    try {
+      sqlContext.createDataFrame(sc.parallelize(TestUtils.expectedData), TestUtils.testSchema)
```

Comment:
Doesn't have to be in this PR, but in general I think I prefer tests that declare their own data, using  `Seq[TupleN].toDF()`.  This is concise, and keeps the data near the test so you can spot problems without needing to cross reference.